testsuite/compat/src/test/java/org/jboss/as/test/compat/jpa/hibernate/transformer/VerifyHibernate51CompatibilityPropertyAndJDSEnabledTransformerTestCase.java

```diff
@@ -71,20 +71,25 @@ public static void skipSecurityManager() {
     }
 
     public static class EnableHibernateBytecodeTransformerSetupTask implements ServerSetupTask {
-        private static final ModelNode PROP_ADDR = new ModelNode()
+        private static final ModelNode PROP_ADDR_ENABLETRANSFORMER = new ModelNode()
                 .add("system-property", "Hibernate51CompatibilityTransformer");
-
+        private static final ModelNode PROP_ADDR_TRANSFORMSESSIMPLMETHODS = new ModelNode()
+                .add("system-property","Hibernate51CompatibilityTransformer.sessImplMtds");
         @Override
         public void setup(ManagementClient managementClient, String s) throws Exception {
-            ModelNode op = Operations.createAddOperation(PROP_ADDR);
+            ModelNode op = Operations.createAddOperation(PROP_ADDR_ENABLETRANSFORMER);
+            op.get("value").set("true");
+            managementClient.getControllerClient().execute(op);
+
+            op = Operations.createAddOperation(PROP_ADDR_TRANSFORMSESSIMPLMETHODS);
             op.get("value").set("true");
             managementClient.getControllerClient().execute(op);
             ServerReload.executeReloadAndWaitForCompletion(managementClient.getControllerClient());
         }
 
         @Override
         public void tearDown(ManagementClient managementClient, String s) throws Exception {
```

Comment:
tearDown doesn't unset PROP_ADDR_TRANSFORMSESSIMPLMETHODS. https://github.com/scottmarlow/wildfly/pull/29 fixes it

Reply:
Thanks, scottmarlow#29 is now merged.